sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -702,6 +773,13 @@ private void processPendingErrors() {
         }
     }
 
+    private static boolean isMemoryRunningLevelCritical(int memoryLevel) {
+        return memoryLevel == TRIM_MEMORY_RUNNING_MODERATE
```

Comment:
We return true for moderate and low, but the api name is critical. Is that intended? Can you please explain?